src/Compilers/Core/Portable/DocumentationComments/DocumentationProvider.cs

```diff
@@ -31,18 +31,18 @@ protected DocumentationProvider()
         /// results from another culture if the preferred culture was unavailable.</param>
         /// <param name="cancellationToken">A cancellation token for the search.</param>
         /// <returns>A DocumentationComment.</returns>
-        protected internal abstract string GetDocumentationForSymbol(
+        protected internal abstract string? GetDocumentationForSymbol(
```

Comment:
>string? [](start = 36, length = 7)

Man, this API is a real victim of lack of NRT. Some implementations return string.Empty when nothing is there, some return null.